### PR TITLE
Add new Website YouTubetoMP3

### PR DESCRIPTION
**Pull request by Nightcorelena**
YouTubetoMP3 is a free MP3 Converter that allows you to download mp3/mp4 Videos.
this is my first time i do a presence, i hope i did everything right.

### CODE REVIEW

File: websites/Y/YouTubetoMP3/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    },
+    "service": "YouTubetoMP3",
+    "altnames": ["YouTube Converter"],
+    "description": {
+      "en": "Downloading Youtube Videos"
+    },
+    "url": "youtubetomp3music.com",
+    "version": "1.0.0",
+    "logo": "https://cdn.discordapp.com/attachments/805925040330637332/867383549194928128/Download_1.png",
+    "thumbnail": "https://media.discordapp.net/attachments/805925040330637332/867383901726048287/featured-image-youtube-downloader.png?width=757&height=473",
```
Comment (Bas950) [inline websites/Y/YouTubetoMP3/dist/metadata.json:15]: upload to imgur

File: websites/Y/YouTubetoMP3/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+ {
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+      "name": "Lena the fauli",
+      "id": "680778360871190600"
+    },
+    "service": "YouTubetoMP3",
+    "altnames": ["YouTube Converter"],
+    "description": {
+      "en": "Downloading Youtube Videos"
```
Comment (Bas950) [inline websites/Y/YouTubetoMP3/dist/metadata.json:10]:
```suggestion
      "en": "Convert YouTube to MP3 in 320kbps within seconds."

```

File: websites/Y/YouTubetoMP3/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+ {
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+      "name": "Lena the fauli",
+      "id": "680778360871190600"
+    },
+    "service": "YouTubetoMP3",
+    "altnames": ["YouTube Converter"],
```
Comment (Bas950) [inline websites/Y/YouTubetoMP3/dist/metadata.json:8]:
```suggestion
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix errors (linebreak)

File: websites/Y/YouTubetoMP3/dist/metadata.json (hunk 1 above)
Comment (Nightcorelena) [reply websites/Y/YouTubetoMP3/dist/metadata.json:15]: Sorry uhm can you edit it? i have no idea how xd 


"logo": "https://imgur.com/puEzTVZ",
"thumbnail": "https://imgur.com/0B8MiPN",

Comment (Bas950) [reply websites/Y/YouTubetoMP3/dist/metadata.json:15]: Edit the file in your editor, commit, push, and it should be updated.

Comment (Nightcorelena) [reply websites/Y/YouTubetoMP3/dist/metadata.json]: thx c:

**Review by Slowlife01 — CHANGES_REQUESTED** [review]
Fix line breaks

**Comment by SlowLife1661** [comment]
They are still there

**Comment by SlowLife1661** [comment]
Still same xd, can you try [this](https://app.execeratics.com/LFandCRLFonline/?l=en)?

**Comment by SlowLife1661** [comment]
Hello, we are sorry to say this, but we've decided to **decline** your Pull Request due to the following reasons:
- **Low quality** Presence.
   - Read DOM only once and only show which page user is viewing.
  - Low quality code.
- Site is **not qualified** to have its own Presence.

**Pull request closed without merging**